src/inverseRobot.js

```diff
@@ -24,7 +24,21 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverseRobotNew = {};
+
+  for (const key in robot) {
+    if (inverseRobotNew.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    if (robot.hasOwnProperty(key)) {
+      const newKey = robot[key];
+
+      inverseRobotNew[newKey] = key;
+    }
```

Comment:
```suggestion
    const newKey = robot[key];

    inverseRobotNew[newKey] = key;
```
Ты же уже в цикле, который перебирает `key in robot`